Sources_v3/Controllers/EntityDatabaseObserver.swift

```diff
@@ -139,10 +139,14 @@ class EntityDatabaseObserver<Item, DTO: NSManagedObject> {
     func startObserving() throws {
         try frc.performFetch()
         
-        _item.reset()
+        reset()
         
         frc.delegate = changeAggregator
     }
+    
+    func reset() {
```

Comment:
Made `reset` internal so that we can manually invalidate an observer's item. 
It's needed because `Core Date` doesn't notify the FRC's delegate when the entity's relationship is changed